src/main/java/com/datastax/cdm/properties/KnownProperties.java

```diff
@@ -80,6 +80,7 @@ public enum PropertyType {
     public static final String ORIGIN_WRITETIME_NAMES = "spark.cdm.schema.origin.column.writetime.names";
     public static final String ALLOW_COLL_FOR_WRITETIME_TTL_CALC = "spark.cdm.schema.ttlwritetime.calc.useCollections";
 
+    public static final String ORIGIN_COLUMN_SKIP = "spark.cdm.schema.origin.column.skip";
```

Comment:
```suggestion
    public static final String ORIGIN_COLUMN_NAMES_TO_SKIP = "spark.cdm.schema.origin.column.skip";
```
could be a better option? But, I'd leave that to you